config/puma.rb

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env puma
+
+rackup DefaultRackup
+
+environment ENV['RACK_ENV'] || 'development'
```

Comment:
Yeah we've used `fetch` in [other](https://github.com/alphagov/govspeak/blob/master/lib/govspeak/header_extractor.rb#L22) [places](https://github.com/guidance-guarantee-programme/pension_guidance/blob/master/spec/controllers/concerns/breadcrumbs_controller_concern_spec.rb#L12) too so probably best to stick with that style.